tests/model_fields/test_uuid.py

```diff
@@ -34,6 +35,13 @@ def test_null_handling(self):
         loaded = NullableUUIDModel.objects.get()
         self.assertEqual(loaded.field, None)
 
+    def test_wrong_value(self):
+        with six.assertRaisesRegex(self, ValueError, 'badly formed hexadecimal UUID string'):
```

Comment:
I tend to prefer `self.assertRaisesMessage` as an alias for `six.assertRaisesRegex`, otherwise this change is fine with me (and to backport).